frontend/packages/pipelines-plugin/src/components/pipelines/pipeline-builder/PipelineBuilderForm.scss

```diff
@@ -10,4 +10,10 @@
   &__short-section {
     max-width: 400px;
   }
+  .pf-c-backdrop {
+    background: none;
+  }
+  .pf-c-modal-box {
+    box-shadow: var(--pf-global--BoxShadow--xl);
+  }
```

Comment:
Since Quicksearch modal is draggable, it will remain in the background so remove this style here and add this box-shadow in the quicksearchModalBody.scss.

Reply:
moved it to `QuicksearchModalBody.scss`.